src/test/java/uk/gov/hmcts/darts/retention/validation/PolicyDurationValidatorTest.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.darts.retention.validation;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
+import uk.gov.hmcts.darts.common.exception.DartsApiException;
+
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+class PolicyDurationValidatorTest {
+
+    private PolicyDurationValidator policyDurationValidator;
+
+    @BeforeEach
+    void setUp() {
+        policyDurationValidator = new PolicyDurationValidator();
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"00Y00M01D", "0Y0M1D", "0Y1M0D", "1Y0M0D", "1Y1M0D", "1Y1M1D"})
+    void validateShouldCompleteWithoutExceptionForValidValues(String durationString) {
+        assertDoesNotThrow(() -> policyDurationValidator.validate(durationString));
+    }
+
+    @ParameterizedTest
+    @ValueSource(strings = {"00Y00M00D", "0Y0M0D"})
+    void validateShouldThrowExceptionForInvalidValues(String durationString) {
+        var exception = assertThrows(DartsApiException.class, () -> policyDurationValidator.validate(durationString));
+        assertEquals("Duration too short", exception.getMessage());
+    }
+
```

Comment:
is it worth having a test for completely dodgy input?

Reply:
This is actually the second layer of `duration` validation - the first layer is via openapi which requires the `duration` value to match regex `^\d{1,2}Y\d{1,2}M\d{1,2}D$`, which should exclude dodgy input.

`PolicyDurationValidator` therefore assumes input is per that format, so the test cases are limited to reflect that.

However I have added a comment in the JavaDoc for `PolicyDurationValidator` which states:
```
...
It is assumed the format of the input string matches this pattern through prior validation checks (e.g. at openapi layer).
...
```